indigo_app/static/javascript/indigo/views/document_source_att.js

```diff
@@ -23,7 +23,19 @@
         this.$dropdown = this.$('.source-attachment-list');
         this.$toggle = this.$('.source-attachment-toggle');
         this.iframe = document.getElementById('source-attachment-iframe');
-        this.mime_types = {'application/pdf': true, 'text/html': true};
+        this.docx_mimetypes = {
+          'application/vnd.openxmlformats-officedocument.wordprocessingml.document': true,
+          'application/msword': true,
+          'application/vnd.oasis.opendocument.text': true,
+          'application/rtf' : true,
+          'text/rtf': true,
+        };
```

Comment:
@longhotsummer I'm happy to have this check here because I think the logic is more clear what we are doing. i.e we want the first document that is of these mimetypes only and nothing else

Reply:
cool